src/components/form/Radio/Radio.tsx

```diff
@@ -31,17 +26,13 @@ export const Radio = forwardRef<HTMLDivElement, RadioProps>(
     const [focused, setFocused] = useState(false)
 
     return (
-      <Container
-        ref={ref}
+      //  eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
+      <div
```

Comment:
Since we use tailwind for styling (as in we don't have global styles attached to certain tags), could we replace `div` with `button`? This will enable us to get rid of `jsx-a11y/no-static-element-interactions`.

Reply:
See this commit: [fix: improve semantic by using label instead of div](https://github.com/getlago/lago-front/pull/1923/commits/f279be54bc61257701a1b1d8a8bba494b8a1f522)